code/modules/integrated_electronics/subtypes/smart.dm

```diff
@@ -4,31 +4,369 @@
 /obj/item/integrated_circuit/smart/basic_pathfinder
 	name = "basic pathfinder"
 	desc = "This complex circuit is able to determine what direction a given target is."
-	extended_desc = "This circuit uses a miniturized, integrated camera to determine where the target is.  If the machine \
```

Comment:
Ditto

Reply:
If you ditto'd all of them you'd be here all day... I will be when fixing this.